### PR TITLE
1616 component alertdialog review

### PR DESCRIPTION
**Pull request by andresz1**
## AlertDialog size

**TASK**:  #1616 

### Description, Motivation and Context
This PR align `AlertDialog` with the specs.
- Remove `size` prop support. It should be fixed to `md` and the width should be adapted to the content.
- Add `isNarrow` prop to `Dialog` component
- Update both `Dialog` and `AlertDialog` documentation

### Types of changes
- [ ] 🛠️ Tool
- [ ] 🪲 Bug fix (non-breaking change which fixes an issue)
- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🧾 Documentation
- [ ] 📷 Demo
- [ ] 🧪 Test
- [ ] 🧠 Refactor
- [ ] 💄 Styles

### CODE REVIEW

File: packages/components/alert-dialog/src/AlertDialogContent.tsx [hunk 1]
```diff
@@ -37,7 +38,9 @@ export const AlertDialogContent = forwardRef<AlertDialogContentElement, AlertDia
           ref={ref}
           data-spark-component="alert-dialog-content"
           {...others}
+          className={cx(className, '!w-auto min-w-sz-288')}
```
Comment (andresz1) [inline packages/components/alert-dialog/src/AlertDialogContent.tsx:41]: In the `Dialog` component the width is set to full. I can think of two options here:
- Override it
- Create a new prop for the `Dialog` and make the full width be optional (default to full width)

WDYT ?

Comment (andresin87) [reply packages/components/alert-dialog/src/AlertDialogContent.tsx:41]: 2nd

Comment (andresz1) [reply packages/components/alert-dialog/src/AlertDialogContent.tsx]: 👍🏻

File: packages/components/dialog/src/Dialog.stories.tsx [hunk 2]
```diff
@@ -154,18 +172,14 @@ export const HTMLForm = () => {
             </Dialog.Header>
 
             <Dialog.Body className="flex flex-col gap-lg">
-              {!isAccountCreated ? (
```
Comment (andresz1) [inline packages/components/dialog/src/Dialog.stories.tsx:157]: @Powerplex I simplify this example so they can just copy/paste as placeholder. WDYT ? 🙏🏻

File: packages/components/dialog/src/Dialog.stories.tsx [hunk 3]
```diff
@@ -15,10 +15,46 @@ const meta: Meta<typeof Dialog> = {
...
+          <Dialog.Body>
+            <Dialog.Description>
+              Make changes to your profile here. Click save when you are done.
+            </Dialog.Description>
+
+            <p>Lorem ipsum dolor sit amet</p>
+          </Dialog.Body>
+
+          <Dialog.Footer className="flex justify-end gap-md">
+            <Dialog.Close asChild>
```
Comment (andresz1) [inline packages/components/dialog/src/Dialog.stories.tsx:41]: I showed that `Dialog.Close` could be used

File: packages/components/alert-dialog/src/AlertDialog.doc.mdx [hunk 4]
```diff
@@ -92,38 +92,12 @@ import { AlertDialog } from '@spark-ui/alert-dialog'
...
-Use `onOpenAutoFocus` prop to listen to when focus moves to the less destructive action after opening. Optionally, the autofocus behavior could be prevented by calling `event.preventDefault`.
-
-<Canvas of={stories.OpenFocus} />
-
-### Focus close
-
-Use `onCloseAutoFocus` prop to listen to when focus moves to the trigger after closing. Optionally, the autofocus behavior could be prevented by calling `event.preventDefault`.
-
-<Canvas of={stories.CloseFocus} />
-
```
Comment (andresz1) [inline packages/components/alert-dialog/src/AlertDialog.doc.mdx:126]: This explanation is available on the tabs panel

**Review by Powerplex — APPROVED** [review]

**Review by kikoruiz — APPROVED** [review]

**Pull request merged**